ipynb/android-wloads/Android_YoutTube_Tests.ipynb

```diff
@@ -0,0 +1,500 @@
+    "    \n",
+    "    # Stop FTrace\n",
+    "    te.ftrace.stop()    \n",
+    "\n",
+    "    # Collect and keep track of the trace\n",
+    "    trace_file = os.path.join(exp_dir, 'trace.dat')\n",
+    "    te.ftrace.get_trace(trace_file)\n",
+    "    \n",
+    "    # Parse trace\n",
+    "    tr = Trace(te.platform, exp_dir,\n",
+    "               events=my_tests_conf['ftrace']['events'])\n",
+    "    ta = TraceAnalysis(tr, te.platform)\n",
+    "    \n",
+    "    # return all the experiment data\n",
+    "    return {\n",
+    "        'dir'             : exp_dir,\n",
+    "        'framestats_file' : framestats_file,\n",
+    "        'trace'           : trace_file,\n",
+    "        'ftrace'          : tr.ftrace,\n",
+    "        'ta'              : ta\n",
+    "    }"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Run experiments and collect traces"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2016-03-24 13:42:07,030 INFO    : ------------------------\n",
+      "2016-03-24 13:42:07,031 INFO    : Run workload using performance governor\n",
+      "2016-03-24 13:43:08,840 INFO    : Collected events spans a 44.539 [s] time interval\n",
+      "2016-03-24 13:43:08,841 INFO    : Set plots time range to (0.000000, 44.538849)[s]\n",
+      "2016-03-24 13:43:10,230 INFO    : ------------------------\n",
+      "2016-03-24 13:43:10,232 INFO    : Run workload using sched governor\n",
+      "2016-03-24 13:44:17,329 INFO    : Collected events spans a 45.192 [s] time interval\n",
+      "2016-03-24 13:44:17,329 INFO    : Set plots time range to (0.000000, 45.191945)[s]\n",
+      "2016-03-24 13:44:18,850 INFO    : ------------------------\n",
+      "2016-03-24 13:44:18,851 INFO    : Run workload using powersave governor\n",
+      "2016-03-24 13:45:34,819 INFO    : Collected events spans a 50.636 [s] time interval\n",
+      "2016-03-24 13:45:34,819 INFO    : Set plots time range to (0.000000, 50.636047)[s]\n",
+      "2016-03-24 13:45:36,265 INFO    : ------------------------\n",
+      "2016-03-24 13:45:36,266 INFO    : Run workload using ondemand governor\n",
+      "2016-03-24 13:46:49,669 INFO    : Collected events spans a 45.314 [s] time interval\n",
+      "2016-03-24 13:46:49,670 INFO    : Set plots time range to (0.000000, 45.313919)[s]\n",
+      "2016-03-24 13:46:51,304 INFO    : ------------------------\n",
+      "2016-03-24 13:46:51,305 INFO    : Run workload using interactive governor\n",
+      "2016-03-24 13:47:59,109 INFO    : Collected events spans a 45.830 [s] time interval\n",
+      "2016-03-24 13:47:59,110 INFO    : Set plots time range to (0.000000, 45.829806)[s]\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
This cell can be replaced with something like:

```
import pandas as pd

# Create results DataFrame
data = {}
for governor in confs:
    data[governor] = {}
    for score_name, score in results[governor]['scores'].iteritems():
        data[governor][score_name] = score

df = pd.DataFrame.from_dict(res)
```

Which better format the table.

You can also add another cell to plot the DataFrame
```
df.plot(kind='bar', rot=45, figsize=(16,8),
        title='PCMark scores vs SchedFreq governors');
```